bril-txt/briltxt.py

```diff
@@ -29,16 +30,17 @@
 eop.2: op ";"
 label.1: LABEL ":"
 
-op: IDENT (FUNC | LABEL | IDENT)*
+op: IDENT (FUNC | LABEL | IDENT | SIGNED_INT)*
 
 ?tyann: ":" type
 
 lit: SIGNED_INT  -> int
   | BOOL         -> bool
   | SIGNED_FLOAT -> float
 
-type: IDENT "<" type ">" -> paramtype
-    | IDENT              -> primtype
+type: IDENT "<" type ("," type)* ">" -> paramtype
+    | IDENT "<" ">"                  -> paramtype
```

Comment:
Seems legit, but maybe this grammar could be a little simpler to allow "zero or more" types? (As in other locations, such as function arguments, where we allow zero or more of things.)